src/pages/interviews/maintainable.mdx

```diff
@@ -17,3 +17,64 @@ Robby speaks with Swizec Teller, Senior Software Engineer at Tia and author of S
 ></iframe>
 
 [Maintainable Podcast – Nov 8, 2021](https://maintainable.fm/episodes/swizec-teller-sr-engineer-mindset)
+
+
+####################
+Talking Points
+####################
+
+* Charateristics of well-maintained software
+* The role of Technical Leadership
+* Swizec's take on technical debt
+* Strong software development opinions Swizec changed his mind on
+* Industry trends Swizec is skeptical of
+* Team Rewrite or Team Refactor?
+* Best practises for working with serverless
+* Traits of Senior Level Engineers
+
+
+####################
+Quotable Quotes
+####################
+
+"The thing that a lot of software engineers really like to forget is that well-maintained software is not just something that follows some textbook idea of perfection. First and foremost, it does have to solve an actual business problem that real people have" - Swizec Teller
+
+"It would be so much easier to develop software if there were no users. And if there were no businesses who need to pay for it, because then you could just make it perfect and amazing." - Swizec Teller
+
+"The older I get, the more, my idea of maintainable software has shifted towards anything that's easy to change and update later because it's probably not gonna live for that long." - Swizec Teller
+
+"I think you continuously have to kind of go through this process of rewriting and updating your software. So you're kind of always in a rewrite state. That's what I've seen works best, at least on the web. I would not suggest that if you're building control systems for cars" - Swizec Teller
+
+"It's very important, especially in modern product companies to have technical leadership, somebody who actually understands how products are built, but that technical leadership needs to have the battle scars and the experience to know when it's worth going for perfection and when it isn't" - Swizec Teller
+
+"A very interesting example from the space industry, even the Mars Rover usually launch towards Mars without software for landing, because 'hey, it's gonna take seven months to get there. That means we have seven more months to write the software. We don't need it yet.'" - Swizec Teller
+
+"What usually happens is that if you're in a startup and you have time to just focus on technical debt, that's usually a bad sign because it means the business has stopped innovating" - Swizec Teller
```

Comment:
```suggestion
"If you're in a startup and you have time to just focus on technical debt, that's usually a bad sign because it means the business has stopped innovating" - Swizec Teller
```